system/settings/settings.xml

```diff
@@ -1821,38 +1821,64 @@
     <category id="upnp" label="20187" help="36322">
       <requirement>HAS_UPNP</requirement>
       <group id="1" label="16000">
-        <setting id="services.upnpserver" type="boolean" label="21360" help="36323">
+        <setting id="services.upnp" type="boolean" label="1270">
```

Comment:
like mentioned on Slack, it's the wrong label ID (it's the one from AirPlay). Correct ID is `21358`.